lib/internal/webstorage.js

```diff
@@ -27,9 +23,9 @@ ObjectDefineProperties(module.exports, {
         const location = getOptionValue('--localstorage-file');
 
         if (location === '') {
-          throw new ERR_INVALID_ARG_VALUE('--localstorage-file',
-                                          location,
-                                          'is an invalid localStorage location');
+          process.emitWarning('`--localstorage-file` was provided without a valid path');
+
+          return undefined;
         }
```

Comment:
I think if we are making the `localStorage` global by default, this check should be deferred. We can already see by some of the test changes here that having this check in the getter for `localStorage` will cause spurious warnings to be emitted even in code that never intentionally touches `localStorage`. This warning should only be emitted when someone actually tries to intentionally use `localStorage`.